bakery/env/prod.json

```diff
@@ -15,7 +15,7 @@
   "GDOC_GOOGLE_FOLDER_ID": "",
   "GDOC_QUEUE_FILENAME": "",
   "GDOC_FEED_FILE_URL": "",
-  "GH_SECRET_CREDS": "((github-username)):((github-api-token))",
+  "GH_SECRET_CREDS": "((github-api-token))",
```

Comment:
Oh, should we update the `bakery/README.md` to not include the `username:` as well?

Reply:
it's not wrong to, but it seems like it might be unnecessary as you noted.

Reply:
Just confirmed it works with personal access tokens, not just api keys (i don't know if they are different tbh)